mautrix/bridge/commands/admin.py

```diff
@@ -29,3 +29,92 @@ async def set_power_level(evt: CommandEvent) -> EventID:
     except (MatrixRequestError, IntentError):
         evt.log.exception("Failed to update power levels")
         return await evt.reply("Failed to update power levels (see logs for more details)")
+
+@command_handler(needs_admin=True, needs_auth=False, name="set-avatar",
+                 help_section=SECTION_ADMIN, help_args="<_mxc_uri_> [_mxid_]",
+                 help_text="Sets an avatar for a ghost user.")
+async def set_ghost_avatar(evt: CommandEvent) -> EventID:
+    try:
+        mxc_uri = evt.args[0]
+    except (KeyError, IndexError):
+        return await evt.reply("**Usage:** `$cmdprefix+sp set-avatar <mxc_uri> [mxid]`")
+    if not mxc_uri.startswith("mxc://"):
+        return await evt.reply("The URI has to start with mxc://.")
+    if len(evt.args) > 1:
+        puppet = await evt.processor.bridge.get_puppet(evt.args[1])
+        if puppet is None:
+            return await evt.reply("The given mxid was not a valid ghost user.")
+        intent = puppet.intent
+    elif evt.is_portal:
+        portal = await evt.processor.bridge.get_portal(evt.room_id)
+        intent = portal.main_intent
```

Comment:
This condition (in all the commands) should error if `intent == evt.az.intent`, there's the config for changing the bridge bot profile

Reply:
Oh I didn't know that the main_intent of a portal can also be the bridge bot, I tried to avoid this by checking for portal room. Will add the check to all commands